salt/version.py

```diff
@@ -96,107 +96,107 @@ class SaltStackVersion(object):
         'Nitrogen'      : (MAX_SIZE - 102, 0),
         'Oxygen'        : (MAX_SIZE - 101, 0),
         'Fluorine'      : (MAX_SIZE - 100, 0),
+        'Neon'          : (MAX_SIZE - 99, 0),
```

Comment:
Just FYI, this is something we have to do each release cycle so that we can have release n+2 supported in ``salt.utils.warn_until()``.